nss-day-cohort-60/django-taters-api
# Lh createcat

Closes #26 


# Description of Proposed Changes
Added create method to category_view

---

# Steps to Test

1. Fetch the lh-createcat branch and switch to it
2. Start/restart debugger
3. Open Postman & add Authorization key with value of Token 0be249c88238743e5b4a7ac370b5145730c28e20
4. Make a POST request to http://localhost:8000/categories { "label": "potato" }
5. GET http://localhost:8000/categories & verify that the new category has been added to the list
